### ISSUE TITLE
`convox system update` error

### ISSUE THREAD

**Issue by nikolay** [convox/cli#79]
```
$ convox system update
ERROR: ValidationError: TemplateURL must be an Amazon S3 URL.
	status code: 400, request id: [xxx-xxx-xxx-xxx-xxx]
```

**Comment by nzoschke** [comment]
Thanks for the report. Fixed in e6d520be

**Comment by nikolay** [comment]
@nzoschke Thanks! I confirm, it works with the latest CLI!
